vendor/github.com/opencontainers/runc/libcontainer/system/linux.go

```diff
@@ -134,3 +134,14 @@ func RunningInUserNS() bool {
 func SetSubreaper(i int) error {
 	return unix.Prctl(PR_SET_CHILD_SUBREAPER, uintptr(i), 0, 0, 0)
 }
+
+// GetSubreaper returns the subreaper setting for the calling process
+func GetSubreaper() (int, error) {
```

Comment:
Same question: How does this fix work? Is this function called somewhere?

Reply:
its called in runc